src/ble/BleLayer.cpp

```diff
@@ -308,6 +308,15 @@ void BleLayer::Shutdown()
 
 void BleLayer::CloseAllBleConnections()
 {
+    //Cancel any ongoing attempt to establish new BLE connection
+    if (mConnectionDelegate != nullptr) {
+        CHIP_ERROR err = mConnectionDelegate->CancelConnection();
+        if (err == CHIP_ERROR_NOT_IMPLEMENTED)
+        {
+            ChipLogDetail(Ble, "BleConnectionDelegate::CancelConnection is not implemented.");
```

Comment:
Why only log this? It seems we should log anything except CHIP_NO_ERROR using err.Format(). Eventually we can approve-list some and explain why they are ignored if we choose so.

Reply:
I will change my code to call CancelBleIncompleteConnection() instead. And I will log any CHIP error returned by CancelBleIncompleteConnection().